src/main/java/gregtech/common/terminal/app/prospector/ProspectingTexture.java

```diff
@@ -44,17 +44,39 @@ public ProspectingTexture(int mode, int radius, boolean darkMode) {
     }
 
     public void updateTexture(PacketProspecting packet) {
+
         int playerChunkX = packet.posX >> 4;
         int playerChunkZ = packet.posZ >> 4;
-        playerI = packet.posX - (playerChunkX - this.radius + 1) * 16 - 1;
-        playerJ = packet.posZ - (playerChunkZ - this.radius + 1) * 16 - 1;
+        playerXGui = packet.posX - (playerChunkX - this.radius + 1) * 16;
```

Comment:
updateTexture is only called while the fluid map is being loaded
so the player position will not receive update after the load is complete

https://user-images.githubusercontent.com/39122497/192614438-4679b417-adb1-4554-857f-6e9d106443f7.mp4